src/sql/workbench/parts/objectExplorer/browser/serverGroupDialog.ts

```diff
@@ -309,13 +310,13 @@ export class ServerGroupDialog extends Modal {
 	// update UI elements that have derivative behaviors based on other state changes
 	private updateView(): void {
 		// check the color buttons and if their checked state does not match the view model state then correct it
-		for (let i = 0; i < this._colorCheckBoxesMap.length; i++) {
-			let { checkbox, color } = this._colorCheckBoxesMap[i];
-			if ((this._viewModel.groupColor === color) && (checkbox.checked === false)) {
-				checkbox.checked = true;
+		for (let i = 0; i < this._colorColorBoxesMap.length; i++) {
+			let { colorbox: colorbox, color } = this._color	esMap[i];
```

Comment:
>	 [](start = 50, length = 1)

It looks like a typo here - there's a tab where I assume there should be _colorColorBoxes